ingestion/src/metadata/utils/constants.py

```diff
@@ -98,3 +98,32 @@
     User.__name__: "user",
     Team.__name__: "team",
 }
+
+# Inverse of ENTITY_REFERENCE_TYPE_MAP
+ENTITY_REFERENCE_CLASS_MAP = {
+    # Service Entities
+    "databaseService": DatabaseService,
+    "messagingService": MessagingService,
+    "dashboardService": DashboardService,
+    "pipelineService": PipelineService,
+    "storageService": StorageService,
+    "mlmodelService": MlModelService,
+    "metadataService": MetadataService,
+    "searchService": SearchService,
+    # Data Asset Entities
+    "table": Table,
+    "storedProcedure": StoredProcedure,
+    "database": Database,
+    "databaseSchema": DatabaseSchema,
+    "dashboard": Dashboard,
+    "dashboardDataModel": DashboardDataModel,
+    "pipeline": Pipeline,
+    "chart": Chart,
+    "topic": Topic,
+    "searchIndex": SearchIndex,
+    "mlmodel": MlModel,
+    "container": Container,
+    # User Entities
+    "user": User,
+    "team": Team,
+}
```

Comment:
Looks good, but might be better to change the `ENTITY_REFERENCE_TYPE_MAP` to avoid having the "same" information duplicated?

I think something like

```python
ENTITY_REFERENCE_TYPE_MAP  = { v.__name__: k for k,v in ENTITY_REFERENCE_CLASS_MAP.items() }
```

could work, what do you think?

Reply:
tried to make it explicit so that it was an easy to read constant, but let's change it back, thanks

Reply:
makes sense, thank you. went too explicit